sklearn/ensemble/_stacking.py

```diff
@@ -967,7 +967,12 @@ def fit(self, X, y, sample_weight=None):
             Returns a fitted instance.
         """
         _raise_for_unsupported_routing(self, "fit", sample_weight=sample_weight)
-        y = column_or_1d(y, warn=True)
+        try:
+            # Single Target
+            y = column_or_1d(y, warn=True)
+        except ValueError:
+            # Multioutput target.
+            y = self._validate_data(X="no_validation", y=y, multi_output=True)
```

Comment:
I think we aren't using try and except to handle this anywhere. Instead maybe we could do something like
```suggestion
        y = self._validate_data(y=y, multi_output=True)
        if y.ndim == 2 and y.shape[1] == 1:
            y = column_or_1d(y, warn=True)
```

Reply:
@OmarManzoor Thank you for review. I have committed your suggestions.